BuildAddOn.py

```diff
@@ -37,6 +37,11 @@ def PrepareParameters (args):
     elif platform.system () == 'Darwin':
         platformName = 'MAC'
 
+    # Load DevKit download data
+    devKitDataPath = pathlib.Path (__file__).absolute ().parent / 'APIDevKitLinks.json'
+    devKitDataFile = open (devKitDataPath)

```

Comment:
The `close` method is never called on the file object. The more "pythonic" way is to use `with` that will close the file object at the end of the scope:
```py
with open (devKitDataPath, 'r') as devKitDataFile:
    devKitData = json.load (devKitDataFile)
```

Reply:
Fixed